spec/locale.html

```diff
@@ -27,7 +27,6 @@ <h1>Intl.Locale ( _tag_ [ , _options_ ] )</h1>
           1. Let _tag_ be ? ToString(_tag_).
         1. Set _options_ to ? CoerceOptionsToObject(_options_).
         1. If IsStructurallyValidLanguageTag(_tag_) is *false*, throw a *RangeError* exception.
-        1. Set _tag_ to CanonicalizeUnicodeLocaleId(_tag_).
```

Comment:
Removing canonicalisation is still a normative change: https://github.com/tc39/ecma402/pull/804#pullrequestreview-1580043877